core/json/variables.examples.json

```diff
@@ -21,14 +21,15 @@
 		"map_center_lat"	:	"50.844441",
 		"map_center_long"	:	"4.363557",
 		"zoom_level"		:	"13",
-		"GMaps_Key"		:	"#GMAPS_KEY#",
-		"timezone"		:	"Europe/Paris",
+		"GMaps_Key"			:	"#GMAPS_KEY#",
+		"timezone"			:	"Europe/Paris",
 		"max_pokemon"		:	151,
 		"pokeimg_suffix"	:	"_flat.png",
 		"mythic_recents"	:	true,
 		"recents_show_iv"	:	false,
 		"captcha_support"	:	false,
 		"iv_numbers"		:	false,
+		"min_rating"		:	9,
```

Comment:
Can you rename this to `recents_min_rating` and put it below `recents_show_iv` please?